cetmix_tower_server/models/cx_tower_variable.py

```diff
@@ -18,6 +18,11 @@ class TowerVariable(models.Model):
     value_ids_count = fields.Integer(
         string="Value Count", compute="_compute_value_ids_count", store=True
     )
+    option_ids = fields.One2many(
+        comodel_name="cx.tower.variable.option",
+        inverse_name="variable_id",
+        string="Options",
```

Comment:
you can add "auto_join=True" here because both models have the same access rights, so we can use 'join' queries without any security issues.

Reply:
I will add